doc/conf.py

```diff
@@ -350,8 +350,8 @@ def setup(app):
     'min_reported_time': min_reported_time,
     'binder': {'org': 'sphinx-gallery',
                'repo': 'sphinx-gallery.github.io',
-               'url': 'https://mybinder.org',
                'branch': 'master',
+               'binderhub_url': 'https://mybinder.org',
```

Comment:
Are we breaking backward compatibility?

Reply:
currently we are not: https://github.com/sphinx-gallery/sphinx-gallery/pull/439/files#diff-2983a963fa400530c600da28315ae7b7R208

should we put in the changelog that the deprecation is scheduled for version XX?